avpipe.c

```diff
@@ -182,7 +184,8 @@ in_read_packet(
         inctx->read_pos += r;
 
         if (inctx->read_bytes - inctx->read_reported > BYTES_READ_REPORT) {
-            in_stat(opaque, in_stat_bytes_read);
+            /* Pass stream_index 0 (stream_index has no meaning for in_stat_bytes_read) */
+            in_stat(opaque, 0, in_stat_bytes_read);
```

Comment:
Should we use -1 for "no stream"?  So future code doesn't think it is stream 0
Or #define a "No stream" value